cli/asc.json

```diff
@@ -390,5 +390,7 @@
   "-O0z": { "value": { "optimizeLevel": 0, "shrinkLevel": 2 } },
   "-O1z": { "value": { "optimizeLevel": 1, "shrinkLevel": 2 } },
   "-O2z": { "value": { "optimizeLevel": 2, "shrinkLevel": 2 } },
-  "-O3z": { "value": { "optimizeLevel": 3, "shrinkLevel": 2 } }
+  "-O3z": { "value": { "optimizeLevel": 3, "shrinkLevel": 2 } },
+  "-Ospeed": { "value": { "optimizeLevel": 3, "shrinkLevel": 0 } },
+  "-Osize": { "value": { "optimizeLevel": 0, "shrinkLevel": 2, "converge": true } }
```

Comment:
Hmm, does `optimizeLevel: 0` really beter than `optimizeLevel: 3` for size shrinking?

Reply:
Would assume it depends. If there is a lot that can be inlined for example, it would depend on how well it optimizes away afterwards. If it doesn't, not inlining in the first place may be better.

Reply:
The `std/array` test produces exactly the same binary size with both `-O3z` and `-Oz`. The bootstrapped compiler seems to have a difference of a couple bytes: 838.188 with `-O3z` vs 838.155 with `-Oz`.

Reply:
I see. Thanks for info!

Reply:
How about make alias for `-Ospeed` and `-O3` just use `-O`?